src/Language/PureScript/CoreFn/Desugar.hs

```diff
@@ -209,6 +212,7 @@ findQualModules decls =
   fqValues :: A.Expr -> [ModuleName]
   fqValues (A.Var q) = getQual' q
   fqValues (A.Constructor q) = getQual' q
+  fqValues (A.TypeClassDictionaryConstructorApp C.IsSymbol _) = getQual' C.IsSymbol
```

Comment:
Perhaps it's not great to explicitly mention `IsSymbol` here: it means we'd have to update multiple places if we're to do a similar instance solving in the future.  But it's possibly odd to do it for all constructor applications?